libraries/standard/mqtt/src/mqtt_lightweight.c

```diff
@@ -57,11 +68,37 @@
  */
 #define MQTT_DISCONNECT_REMAINING_LENGTH    ( ( uint8_t ) 0 )
 
+/*
+ * Constants relating to CONNACK packets, defined by MQTT 3.1.1 spec.
+ */
+#define MQTT_PACKET_CONNACK_REMAINING_LENGTH        ( ( uint8_t ) 2U )    /**< @brief A CONNACK packet always has a "Remaining length" of 2. */
+#define MQTT_PACKET_CONNACK_SESSION_PRESENT_MASK    ( ( uint8_t ) 0x01U ) /**< @brief The "Session Present" bit is always the lowest bit. */
+
+/**
+ * @brief The size of MQTT PUBACK, PUBREC, PUBREL, and PUBCOMP packets, per MQTT spec.
+ */
+#define MQTT_PUBLISH_ACK_PACKET_SIZE        ( 4U )
+
```

Comment:
Ditto. These defines just end up adding a lot of lines and noise to the file which is not adding value.